### PR TITLE
Add web UI > Organizations content

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/16473

This PR adds the following pages under `UIs and APIs` > `Web user interface`:

- Organization management
  - Creating, inspecting, deleting
  - Access control

Also a shortcode `organization_mgmt_mapi_disclaimer` is added. Preview:

image

## TODO

- [x] Consistently use `Management API` spelling
- [x] Clarify whether customers will be able to create own Organization CRs. Adapt content accordingly.
- [x] Update content to reflect the situation that non-admin users will not yet be permitted in the next stage.

### CODE REVIEW

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:4]: 'organization access control' is a bit of a mouthful - maybe something like 'for organization management'?

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:27]:
```suggestion
The access control UI in the web interface provides quick access to a focused set of RBAC resources, with the goal to make it easy for you to give someone – be it individuals or groups – or service accounts access to resources in an organization's namespace.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:42]: > A list of roles, both from the organization's namespace and cluster roles, to select from.

what do you think about

> A list of roles to select from, sourced from both the organization's namespace and cluster roles

? to avoid the embedded clause (which i always find challenging as a reader, especially for technical content)

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:64]:
```suggestion
The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering these icons with your pointer will reveal additional information in a tooltip.
```

**Review by JosephSalisbury — COMMENTED** [review]
have a lil' first pass

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:42]: Good!

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:4]: Going for "organization RBAC", since "organization management" is more the broader context here.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:64]:
```suggestion
The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering over these icons with your pointer will reveal additional information in a tooltip.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:124]: > upper-

what's with the hypen here?

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:126]:
```suggestion
**Note:** Your Giant Swarm account engineer (AE) is happy to assist in case there are uncertainties about the correct spelling.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:26]:
```suggestion
In the web user interface top navigation menu you can easily find the **Organizations** section. Click this item to navigate to an overview of the organizations you have access to.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:32]:
```suggestion
To learn more about an individual organization, and to find more management functions, click on one of the organizations in the list.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:53]:
```suggestion
As a permitted user - either an admin or a non-admin user with _delete_ permission for the [Organization]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md">}}) resource - you will find at the bottom of this page a section titled **Delete this organization**.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:124]: Oh, my bad transfer from German language.

Comment (marians) [inline src/content/ui-api/web/organizations/access-control/index.md:124]:
```suggestion
When adding a **group** or a **user**, you have to enter the exact user or group identifier as your identity provider presents them. This includes upper and lowercase spelling. User identifiers may or may not be email addresses, according to your set-up.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (axbarsan) [inline src/content/ui-api/web/organizations/access-control/index.md:25]:
```suggestion
Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the Management API, including via the web user interface itself.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (axbarsan) [inline src/content/ui-api/web/organizations/access-control/index.md:52]:
```suggestion
While the **Subjects** tab will be explained in more detail below under [granting access](#granting-access), the information shown under **Permissions** deserves a closer look here.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (pipo02mix) [inline src/content/ui-api/web/organizations/access-control/index.md:150]: I am putting myself on customer shoes and I see they would like to remove a user from a certain group. I know this means removing the user from IdP but I would add a warning/note here explaining the case

Comment (pipo02mix) [reply src/content/ui-api/web/organizations/access-control/index.md:150]: Maybe the same for the addition of a certain user to a group in IdP (like "adding a user in the same user group on your IdP connector will result in assigning permissions automatically without any further action in our platform")

**Review by axbarsan — APPROVED** [review]
LGTM in general

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [reply src/content/ui-api/web/organizations/access-control/index.md]: ezzz

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:55]:
```suggestion
Apart from adequate permission, in order to be able to delete an organization there must not be any workload clusters owned by the organization.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:51]:
```suggestion
From the organizations overview, click on an organization in order to open its detail page.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:40]:
```suggestion
- Must be unique within the management cluster.
```

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:30]:
```suggestion
If you are an admin user, this page shows all organizations existing in the management cluster of your installation. As a non-admin user, you see the organizations you have access to.
```

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:128]:
```suggestion
You can enter or paste multiple identifiers, separated by a space or comma, into a single input field.
```

**Review by JosephSalisbury — APPROVED** [review]
just a few nits, but let's roll

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:150]: Sounds reasonable, will add something.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:25]: I'll have to get this sorted everywhere. It's not consistent currently. Thanks

File: src/content/ui-api/web/app-platform/index.md
Comment (Oshratn) [inline src/content/ui-api/web/app-platform/index.md:4]: I'm confused about our usage of App Platform and App Catalog/Managed Apps. Isn't App Platform a set of capabilities that enables App Catalog/Managed Apps?
If so, I recommend changing the wording. Ping @cokiengchiara / @rossf7

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (Oshratn) [inline src/content/ui-api/web/organizations/access-control/index.md:64]:
```suggestion
The single letter icons in the **Verbs** column each represent one of the seven standard verbs: `get`, `watch`, `list`, `create`, `update`, `patch`, and `delete`. The last items, showing an ellipsis `⋯`, stands for additional, arbitrary verbs. Hovering over these icons with your pointer will reveal additional information in a tooltip.
```
I'm not sure what arbitrary means here. I'm pretty sure you didn't mean random here. Maybe we can find a less arbitrary word 😉

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (Oshratn) [inline src/content/ui-api/web/organizations/access-control/index.md:126]: I'd switch 'spelling' with 'format', since I don't think the actual spelling is the problem here. I'm under the impression that it's more about case usage, etc.

**Review by Oshratn — CHANGES_REQUESTED** [review]
Adding a general comment too.
Since you are touching the documents, you can also update the last review date field.

**Review by pipo02mix — APPROVED** [review]
LGTM when Oshrat is happy

File: src/content/ui-api/web/app-platform/index.md
Comment (marians) [reply src/content/ui-api/web/app-platform/index.md:4]: That's not part of this change.

I would update last_review_date if I actually went over these entire documents. This PR here is about adding pages.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:64]: Kubernetes allows to use any string as a "verb" for permissions. The verbs are meant as in `<subject> can to <verb> to <object>`. It is completely up to the user to invent things like `run`, `check` or `banana`.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:126]: Will change

**Comment by marians** [comment]
@Oshratn With the changes I made, can you un-block the PR?

File: src/content/ui-api/web/app-platform/index.md
Comment (rossf7) [reply src/content/ui-api/web/app-platform/index.md:4]: @marians Yes this PR isn't changing the text but @Oshratn I think it's correct as is.

> In short: the Giant Swarm App Platform refers to the whole feature, and an App Catalog is a collection of Apps.

This is from https://docs.giantswarm.io/app-platform/overview/ and there is more explanation there.

Comment (Oshratn) [reply src/content/ui-api/web/app-platform/index.md:4]: Unblocking the PR. 

@rossf7 can we work on separate PR to clarify the language around App Platform and App Catalog?

**Review by Oshratn — APPROVED** [review]

**Comment by marians** [comment]
This has been updated once more and is now considered to be ready. If you want to give it a final review, this is your chance.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:27]:
```suggestion
The access control UI in the web interface provides quick access to a focused set of RBAC resources, with the goal to make it easy for you to give individuals, groups, or service accounts access to resources in an organization's namespace.
```

does that read a little easier? i don't love the current construction, but it's hard to find something better

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/access-control/index.md:31]: > management cluster/installation

could we be a bit more specific here?

File: src/content/ui-api/web/organizations/basic-management/index.md
Comment (JosephSalisbury) [inline src/content/ui-api/web/organizations/basic-management/index.md:50]: Don't we do this now?

Comment (marians) [reply src/content/ui-api/web/organizations/basic-management/index.md:50]: No, on KVM we are practically blind from the happa and MAPI perspective.

File: src/content/ui-api/web/organizations/access-control/index.md
Comment (marians) [reply src/content/ui-api/web/organizations/access-control/index.md:31]: Not yet. We have a little widget planned to help with this (https://github.com/giantswarm/giantswarm/issues/16876) but it's not there yet.

**Pull request merged**
